src/tokens/transformation/android/formats/fontStyle.js

```diff
@@ -19,15 +19,17 @@ module.exports = ({ dictionary, platform, options = {}, file }) => {
           compositeToken.name.replace('font_android_', '')
         )}">\n` +
         printDescription(compositeToken.description) +
-        `    <item name="android:fontFamily">${compositeToken.original.value.fontFamily}</item>\n` +
+        `    <item name="android:fontFamily">@font/${changeCase.snakeCase(compositeToken.original.value.fontFamily)}</item>\n` +
         `    <item name="android:textSize">@dimen/${changeCase.snakeCase(
           compositeToken.name.replace('font_android_', '')
         )}</item>\n` +
-        `    <item name="android:lineHeight">${compositeToken.original.value.lineHeight}sp</item>\n` +
-        `    <item name="android:letterSpacing">${letterSpacingToFloat(
-          compositeToken.original.value.letterSpacing,
-          compositeToken.original.value.fontSize
-        )}</item>\n` +
+        // for android:lineHeight requires API level 28 and current api is 26
+        `    <item name="lineHeight">${compositeToken.original.value.lineHeight}sp</item>\n` +
```

Comment:
should this be commented out too if it's unsupported on our Android API version?

Reply:
@fallaciousreasoning attribute with `android:` is not supported yet but we can use `lineHeight` with existing apis.